io_scene_godot/__init__.py

```diff
@@ -46,11 +46,11 @@ class ExportGodot(bpy.types.Operator, ExportHelper):
     bl_options = {"PRESET"}
 
     filename_ext = ".escn"
-    filter_glob = StringProperty(default="*.escn", options={"HIDDEN"})
+    filter_glob : StringProperty(default="*.escn", options={"HIDDEN"})
```

Comment:
could you explain why this change?

Reply:
Blender 2.8 on loading produces warnings in the console if you don't set properties using 'annotations' (colons instead of equal signs), as in this:
https://github.com/KhronosGroup/glTF-Blender-IO/issues/147

Reply:
> Thanks!
> 
> May I cherry-pick your commits to the #214? the `to_mesh_clear` is exactly what I miss in that PR.

Sure. I've made further improvements since submitting the pull request though.
The 'Apply Modifiers' option was doing nothing, and no modifiers were being applied to meshes if it was checked.
Not all that familiar with github, if I push the new commit to my fork do I need to submit a new pull request?

Reply:
You are doing correct, just push commits to your fork and github would update PR